src/index.js

```diff
@@ -0,0 +1,88 @@
+import { helpers } from '@eik/common';
+import { join } from 'path';
+import fetch from 'node-fetch';
+
+const isUrl = (value = '') => value.startsWith('http');
+
+const fetchImportMaps = async (urls = []) => {
+    try {
+        const maps = urls.map((map) => fetch(map).then((result) => {
+            if (result.status === 404) {
+                throw new Error('Import map could not be found on server');
+            } else if (result.status >= 400 && result.status < 500) {
+                throw new Error('Server rejected client request');
+            } else if (result.status >= 500) {
+                throw new Error('Server error');
+            }
+            return result.json();
+        }));
+        return await Promise.all(maps);
+    } catch (err) {
+        throw new Error(
+            `Unable to load import map file from server: ${err.message}`,
+        );
+    }
+};
+
+export default class EikNodeClient {
+    constructor({
+        development = false,
+        base = '',
+        path = process.cwd(),
+    } = {}) {
+        this.pDevelopment = development;
+        this.pConfig = {};
+        this.pPath = path;
+        this.pBase = base;
+        this.pMaps = [];
+    }
+
+    async load({
+        maps = false,
```

Comment:
Regarding your questions, I agree that `maps` should be a flag rather than a separate method. As far as where it should be defined, I think a constructor flag makes the most sense unless the load method is likely to be called more than once for different purposes (which I wouldn't think so?).